internal/operator/cluster/cluster.go

```diff
@@ -482,6 +591,53 @@ func UpdateTablespaces(clientset kubernetes.Interface, restConfig *rest.Config,
 	return nil
 }
 
+// annotateBackrestSecret annotates the pgBackRest repository secret with relevant cluster
+// configuration as needed to support bootstrapping from the repository after the cluster
+// has been deleted
+func annotateBackrestSecret(clientset kubernetes.Interface, cluster *crv1.Pgcluster) error {
+
+	clusterName := cluster.GetName()
+	namespace := cluster.GetNamespace()
+
+	// simple helper that takes two config options, returning the first if populated, and
+	// if not the returning the second (which also might now be populated)
+	cfg := func(cl, op string) string {
```

Comment:
Not sure why this func is embedded, might as well be at the top level

Reply:
Really I was just trying to create some terse syntax purely in support of populating map below (e.g. vs. a bunch of if/else's for each setting like we have in other places).

I can def move it up a level though.  Figured this one might lead to a question :)

Reply:
Well I had moved it up a level, but now not sure ... :grin:

Reply:
Leaving as a local function for now per additional discussion.